apps/app.go

```diff
@@ -192,16 +198,9 @@ type App struct {
 	GrantedLocations Locations `json:"granted_locations,omitempty"`
 }
 
-func (a *App) ConfigMap() map[string]interface{} {
-	b, _ := json.Marshal(a)
-	var out map[string]interface{}
-	_ = json.Unmarshal(b, &out)
-	return out
-}
-
-func AppFromConfigMap(in interface{}) *App {
-	b, _ := json.Marshal(in)
-	var out App
-	_ = json.Unmarshal(b, &out)
-	return &out
+type MarketplaceApp struct {
```

Comment:
TODO: add comment describing MarketplaceApp  (e.g. what is the difference from `Manifest`).

Reply:
@hanzei introduced this data struct, do you mind writing up a comment about how it is used?